profiler/shakesapp/shakesapp/server.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package shakesapp defines a server which can be queried to determined
+// many times a string appears in the works of Shakespeare, and a client
+// which can be used to send load to that server.
+package shakesapp
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strings"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+// NewServer returns an implementation of the server for ShakespeareService
+// (defined in shakesapp.proto).
+func NewServer() ShakespeareServiceServer {
+	return &server{}
+}
+
+type server struct{}
```

Comment:
Might move this above `NewServer` and give it a comment. Can look a little odd to someone who isn't used to gRPC services.

Reply:
Done.